### ISSUE TITLE
Too many Issues and Pull requests

### ISSUE THREAD

**Issue by ayushlodhi11** [smartinez87/exception_notification#340]
Why are there too many issues and pull requests pending? can we do something about it? Either pull requests should get closed if there is a problem or it should get merged, similar with the issues.

**Comment by san650** [comment]
@ayushlodhi11
> Either pull requests should get closed if there is a problem or it should get merged, similar with the issues.

You're right, but I guess we should give users the possibility to fix the problem by letting them know what it is before closing it.

> similar with the issues.

We should check if the issues are valid or not, if they are features or bug and close the ones that are not reproducible in master.

> can we do something about it?

Bug triage would be great, see if the issues are bugs or features, if they are bugs we should mark them with a `bug` label so we can prioritize them.

**Comment by ayushlodhi11** [comment]
@san650 
Alright, then lets prioritize the issues and fix it asap.
We can ask users to fix the problem in PR but sum pr are too old, I doubt if user will be keen to fix it, anyways we can still give a try and if we don't get response in some time and we can close it.

**Comment by aried3r** [comment]
Any progress on this? I don't really have time to maintain anything, but I could look into PRs regarding the slack notifier that can't be merged without conflicts, like #313.
